erpnext/patches/v7_0/convert_timelog_to_timesheet.py

```diff
@@ -30,7 +30,12 @@ def execute():
 		time_sheet.flags.ignore_validate = True
 		time_sheet.flags.ignore_links = True
 		time_sheet.save(ignore_permissions=True)
-
+		frappe.db.sql("""update tabCommunication 
```

Comment:
This should be a new patch, it should also be run for those accounts where `convert_timelog_to_timesheet.py` has already been executed.

And please send this pull requests to `hotfix` branch.

Thanks for this pull request 👍